### PR TITLE
add unit tests for the events table

### PR DESCRIPTION
**Pull request by hriday-panchasara**
Created some unit tests for user-defined functions and components used in the events table. Was unable to create comprehensive tests for TableOutputComponent since its entirely built upon the AG-Grid library, and snapshot testing does not show the rendered HTML of AG-Grid. Another way to test TableOutputComponent could be to stub api calls made by the component to fetch table data and check if the internal state of the component (table columns & lines) updates. I was unable to mock tsp-client api calls or stub responses since jest could not reference the requests being made by the component's methods. But I believe we can later add to this test once we figure out a way to do so. Also included snapshot and unit tests for TableRendererComponents which are used by TableOutputComponent

Signed-off-by: hriday-panchasara <hriday.panchasara@ericsson.com>

### CODE REVIEW

File: packages/react-components/src/components/__tests__/events-table-component-tests.tsx [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+});
+
+describe('Table output component tests', () => {
+    
+    test('Renders AG-Grid table with provided props & state', async () => {
+        const tableOutputComponentProps: AbstractOutputProps = {
+            tooltipComponent: null,
+            style: {
+                width: 0,
+                chartWidth: 0,
```
Comment (PatrickTasse) [inline packages/react-components/src/components/__tests__/events-table-component-tests.tsx:26]: Sorry for taking so long to review. The AbstractOutputProps have been updated and some of these properties have changed. The unit test fails to compile on latest master.

**Review by PatrickTasse — CHANGES_REQUESTED** [review]

File: packages/react-components/src/components/__tests__/events-table-component-tests.tsx (hunk 1 above)
Comment (hriday-panchasara) [reply packages/react-components/src/components/__tests__/events-table-component-tests.tsx]: @PatrickTasse Rebased on top of master. The unit tests should pass now

**Review by PatrickTasse — APPROVED** [review]

**Review by Rodrigoplp-work — CHANGES_REQUESTED** [review]
Excellent work!

Please change the name of the test file from `event-table-component-tests.tsx` to `table-renderer-components.test.tsx`, and the respective snapshot.

That follows the way we did with the other tests and makes it easier for other developers to identify what has already been covered when writing more tests.

**Comment by hriday-panchasara** [comment]
> Excellent work!
> 
> Please change the name of the test file from `event-table-component-tests.tsx` to `table-renderer-components.test.tsx`, and the respective snapshot.
> 
> That follows the way we did with the other tests and makes it easier for other developers to identify what has already been covered when writing more tests.

@Rodrigoplp-work renamed the relevant files

**Review by PatrickTasse — APPROVED** [review]

**Review by Rodrigoplp-work — APPROVED** [review]

**Pull request merged**